lib/reporters/dot_reporter.js

```diff
@@ -36,13 +37,18 @@ DotReporter.prototype = {
     if (this.silent) {
       return;
     }
+    if (this.currentLineChars > 60) {
```

Comment:
Could you maybe use https://nodejs.org/api/tty.html#tty_writestream_columns to detect length from the terminal?

Reply:
@smottt did you had the chance to take a look?

Reply:
@johanneswuerbach Sorry about the long delay, was quite busy lately. I'll check this at some point this week.